src/java.base/share/classes/java/lang/CharSequence.java

```diff
@@ -302,4 +302,44 @@ public static int compare(CharSequence cs1, CharSequence cs2) {
         return cs1.length() - cs2.length();
     }
 
+    /**
+     * Characters are copied from this sequence into the
+     * destination character array {@code dst}. The first character to
```

Comment:
I think the "Characters are copied ..." sentence goes back to StringBuffer in JDK 1.0. CS doesn't need to copy this exactly and I think better to say that it copies chars from this sequence into the given destination array.

Reply:
Do you like this more: `Copies chars from this sequence into the given destination array.` ?

Reply:
Thank you, Alan. Fixed in https://github.com/openjdk/jdk/pull/21730/commits/a7f56f4ecba4be1a28e551e8a247998db7d7cb79.